src/pages/Integrations.vue

```diff
@@ -0,0 +1,60 @@
+<router lang="yaml">
+  name: WSL Integration
+</router>
+
+<script lang="ts">
+import os from 'os';
+import { ipcRenderer } from 'electron';
+import Vue from 'vue';
+import Integration from '@/components/Integration.vue';
+
+export default Vue.extend({
+  components: { Integration },
+  data() {
+    return {
+      /** @type Record<string, boolean | string> */
+      integrations: {},
```

Comment:
I realize you probably just restored this from an old commit, but this should be (because this is already a TypeScript chunk):
```suggestion
      integrations: {} as Record<string, boolean | string>,
```